### PR TITLE
Adds two new perks for nanogate / Buffs Nanogates costs / Adds more modules to RIG suit

### PR DESCRIPTION
**Pull request by Guidesu**
## About The Pull Request

	About The Pull Request:
</summary>
<hr>
First and foremost. Maintainers like trilby and moon. I BEG YOU to try this values, i tested this value like for 3 hours straight. They look like they are too low but they are not. TM it if you must.

You absolutely can't buy everything and you still need to mind what you buy, but you have way more choice to try new things out. I tested it with the guild nanogate and opifex, everything worked well, i could choose and mix rig suit with personal buffs to nanobot, i could make things fairly upgraded with the rig/bot/me and i used pretty much all the points. The most basic nanogate gets most of this new values. They have more option to try new things out. Opifex and Artificers can buy more stuffs yes but that was always (i think from the OG devs) to be that way. So PLEASE. Trust on me this one and try this value. We can adjust later if needed and if it feels too overpowered.

Nanobot to normal one: All full upgrades for 10 points
11 points for opifex for the food module

RIG suits with new modules: 10 to buy everything in all the variants

User upgrades: 16 for all variants that includes regen / more speed / health / and the new perks
17 for opifex with the module to make ammo.

Opifex even if they fully upgrade something they have can choose to mix it or fully upgrade or not even fully stuffs.

Again. Let's try this value first.

	
<hr>
</details>

## Changelog
:cl:
tweak: Adds two new perks for nanogate / Buffs Nanogates costs / Adds more modules to RIG suit

/:cl:

### CODE REVIEW

File: code/__DEFINES/misc.dm [hunk 1]
```diff
@@ -247,7 +247,7 @@
 #define EMAIL_SYSADMIN  "admin@internal-services.net"
 #define EMAIL_BROADCAST "broadcast@internal-services.net"
 #define EMAIL_PAYROLL "payroll@internal-services.net"
-#define EMAIL_NANOGATE "satellite-uplink@nanogate.populi.net"
+#define EMAIL_NANOGATE "satellite-uplink@nanogate.net"
```
Comment (Trilbyspaceclone) [inline code/__DEFINES/misc.dm:250]: why are you removing this?

File: code/datums/perks/nanogate.dm [hunk 2]
```diff
@@ -59,6 +59,16 @@
 	holder.health -= 40
 	..()
 
+/datum/perk/nanite_power/nanite_armor/on_emp()
```
Comment (Trilbyspaceclone) [inline code/datums/perks/nanogate.dm:62]: why does this deal damage and then heal?

File: code/datums/perks/nanogate.dm [hunk 3]
```diff
@@ -202,3 +212,50 @@
...
+	name = "Nano Purge Augment"
+	desc = "You are augmented with a Nano Purge Augment, a nanite-based mechanism designed to detoxify your system. It rapidly neutralizes toxins and other harmful substances that is highly addictive, ensuring your body and mind remain unaffected by various poisons and drugs."
+	gain_text = "You feel unnatural calmness on your liver."
+	lose_text = "Your start to crave after things again."
+
+/datum/perk/chemical_neutralizer/assign(mob/living/L)
+	..()
+	if(ishuman(holder))
+		var/mob/living/carbon/human/H = holder
+		H.metabolism_effects.addiction_chance_multiplier = 0.05
```
Comment (Trilbyspaceclone) [inline code/datums/perks/nanogate.dm:251]: why are you setting this to 0.05 and then on remove setting it to be the same

File: code/modules/nanogate/powers/rig.dm [hunk 4]
```diff
@@ -140,5 +161,104 @@ List of powers in this page :
...
+		var/obj/item/rig_module/vision/meson/meson_module = new /obj/item/rig_module/vision/meson(src)
+		meson_module.Initialize()
+
+		nanite_rig.install(meson_module)
+		to_chat(owner, "Your spine hurt as the nanites start to work on making an module.")
+		verbs -= /obj/item/organ/internal/nanogate/proc/nanite_rig_meson
+
+/obj/item/organ/internal/nanogate/proc/nanite_rig_power_sink()
+	set category = "Nanogate Powers"
+	set name = "Nanite Rigsuit - Power Sink Module (1)"
```
Comment (Trilbyspaceclone) [inline code/modules/nanogate/powers/rig.dm:207]: this should be locked to opfiex type not just anyone

File: code/modules/nanogate/powers/user.dm [hunk 5]
```diff
@@ -210,9 +210,9 @@ List of powers in this page :
 // Allows you to drink metals for food
 /obj/item/organ/internal/nanogate/proc/nanite_metal_drinker()
 	set category = "Nanogate Powers"
-	set name = "Nanite Augment - Metal-Eater (2)"
+	set name = "Nanite Augment - Metal-Eater (1)" // It is now affected by EMPs
```
Comment (Trilbyspaceclone) [inline code/modules/nanogate/powers/user.dm:213]: No its not? Where is EMPs disabling a perk?

File: code/modules/sanity/sanity_mob.dm [hunk 6]
```diff
@@ -191,7 +191,7 @@ GLOBAL_VAR_INIT(GLOBAL_INSIGHT_MOD, 1)
 		for(var/mob/living/carbon/human/H in view(owner))
 			if(H.sanity.level > 60)
 				moralist_factor += 0.02
-	give_insight((INSIGHT_GAIN(level_change) * insight_passive_gain_multiplier * moralist_factor * life_tick_modifier * GLOB.GLOBAL_INSIGHT_MOD) * (owner.stats.getPerk(PERK_INSPIRED) ? 1.5 : 1) * (owner.stats.getPerk(PERK_NANOGATE) ? 0.4 : 1) * (owner.stats.getPerk(PERK_COGENHANCE) ? 1.1 : 1))
+	give_insight((INSIGHT_GAIN(level_change) * insight_passive_gain_multiplier * moralist_factor * life_tick_modifier * GLOB.GLOBAL_INSIGHT_MOD) * (owner.stats.getPerk(PERK_INSPIRED) ? 1.5 : 1) * (owner.stats.getPerk(PERK_NANOGATE) ? 0.4 : 1) * (owner.stats.getPerk(PERK_COGENHANCE) ? 1.1 : 1) * (owner.stats.getPerk(PERK_NANITE_PYSCHEBOOST) ? 1.05 : 1))
```
Comment (Trilbyspaceclone) [inline code/modules/sanity/sanity_mob.dm:194]: why is the "perk that helps with chemicals" undoing the major downside of the nanogate?

**Comment by Trilbyspaceclone** [comment]
Also please accually document your changes in the pr ya?

**Comment by AysheDaArtist** [comment]
If you need any perk icons let me know and I'll whip em up quick for ya!

**Comment by Guidesu** [comment]
Sorry i made this really quickly rushed and did not put much though on it @Trilbyspaceclone

**Comment by Guidesu** [comment]
> Also please accually document your changes in the pr ya?

Will do

File: code/__DEFINES/misc.dm (hunk 1 above)
Comment (Guidesu) [reply code/__DEFINES/misc.dm:250]: no clue. Don't remember why. Fixed

**Comment by Guidesu** [comment]
> If you need any perk icons let me know and I'll whip em up quick for ya!

I in fact. Do not need your help. Whatsoever.

**Regardless**

I will work on this probably on this week

**Comment by Trilbyspaceclone** [comment]
Merge conflicts + outdated code, the idea is not rejected

**Pull request closed without merging**